pkgs/README.md

```diff
@@ -70,6 +70,7 @@ Because entries in the Nix store are inert and do nothing by themselves, package
   * These aspects are sometimes hard to verify, in which case an upstream that is not known to be irresponsible should be considered as responsible.
 * Source-available software should be built from source where possible.
   Binary blobs risk supply chain attacks and vendored outdated libraries.
+  If you cannot build from source, you will almost always have to set `meta.sourceProvenance = [ lib.sourceTypes.binaryNativeCode ]`.
```

Comment:
I'm not sure this belongs here, as this section is quite focused on security.  If we start adding things that are not strictly about security, it's going to get less readable.

Reply:
Fair enough, I follow that logic. I'll remove that.